x/gateway/genesis.go

```diff
@@ -2,15 +2,16 @@ package gateway
 
 import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
+
 	"pocket/x/gateway/keeper"
 	"pocket/x/gateway/types"
 )
 
 // InitGenesis initializes the module's state from a provided genesis state.
 func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState) {
 	// Set all the gateway
-	for _, elem := range genState.GatewayList {
-		k.SetGateway(ctx, elem)
+	for _, gateway := range genState.GatewayList {
```

Comment:
Addressed in #75 maybe worth undoing these to avoid conflicts

Reply:
I think conflict resolution with a `git merge` will be quite easy.

If this is something you find yourself spending a lot on time on (i.e. > 2 minutes), let me know and I can show you quick ways of getting it done.